hydra-node/test/Hydra/Chain/Direct/StateSpec.hs

```diff
@@ -389,7 +398,7 @@ forAllFanout action =
        in action utxo tx
             & label ("Fanout size: " <> prettyLength (countAssets $ txOuts' tx))
  where
-  maxSupported = 39
+  maxSupported = 30
```

Comment:
this change was not needed and got merged by mistake.

Reply:
Change reverted by [this commit](https://github.com/input-output-hk/hydra/pull/702/commits/2e23df011462e67267b11d9d30e57399dcb6288b)